crates/symbolicator-js/src/lookup.rs

```diff
@@ -931,9 +944,11 @@ impl ArtifactFetcher {
 
                 // Get the sourcemap reference from the artifact, either from metadata, or file contents
                 let sourcemap_url = resolve_sourcemap_url(abs_path, &artifact.headers, &contents);
+                let debug_id = discover_debug_id(&contents);
                 CachedFile {
                     contents: Some(contents),
                     sourcemap_url: sourcemap_url.map(Arc::new),
+                    debug_id,
```

Comment:
I suspect the `test_expansion_with_release_dist_pair_x` test failure where `resolved_with` ends up as `debug-id` rather than the expect `release` is down to this. When calling `try_fetch_file_from_artifacts` I suspect we shouldn't be reading the debug id from the contents and it should just be set to `None`

Reply:
The test failure is caused by the change on line 499, this change has no effect.